### DISCUSSION TITLE
如何表单展示这个字段，但是不提交这个字段的值。

### DISCUSSION THREAD

**Discussion by kinglinggo** [alibaba/formily#3743] [Q&A 2.x]
Json Schema 模式

**Comment by chenWebCoder** [comment]
https://formilyjs.org/zh-CN/guide/advanced/linkages#effects-%E7%94%A8%E4%BE%8B

display: 'visible' | 'none' | 'hidden'

**Reply by kinglinggo** [reply]
visible：显示
none：隐藏
hidden：隐藏-保留值

需求是想要个 显示-不保留值 的设置

**Reply by simplast** [reply]
可以自定义一个组件“Text”，就展示内容即可，内容字段使用除value外的字段

**Reply by chenWebCoder** [reply]
提交时手动过滤

**Comment by yanhao98** [comment]
type: void，应该也可以，不确定。

**Reply by xiaosi-ss** [reply]
这个才是标准用法

**Comment by PlutoCA** [comment]
type: void

**Comment by kinglinggo** [comment]
提交时手动过滤了。设为void UI组件，就没有value属性了，内部的组件值没法渲染

**Reply by yanhao98** [reply]
Void通过prop给value。
